rpm/sdk-setup.spec

```diff
@@ -85,6 +85,7 @@ Requires:   git
 Requires:   tar
 Requires:   p7zip-full
 Requires:   gnu-cpio
+Requires:   compiledb
```

Comment:
Just checking that I understand correctly this change: Instead of running compiledb on the targets, we now run only make (in dry-run mode) in target, and parse the output on build engine side?

Reply:
... reading the commit message would of course help ;) LGTM